src/SignalR/clients/ts/signalr/src/HttpClient.ts

```diff
@@ -162,4 +166,14 @@ export abstract class HttpClient {
     public getCookieString(url: string): string {
         return "";
     }
+
+    /** Gets all headers that apply to requests. */
+    public getHeaders(): MessageHeaders {
```

Comment:
Lets get rid of both of these methods and instead pass the user provided headers to all the transports in the constructors.

Reply:
Fixed: https://github.com/dotnet/aspnetcore/pull/19227/commits/29995db5f0213625085da7a15731c26f68dd859b